src/pages/api/auth/[...nextauth].ts

```diff
@@ -1,15 +1,34 @@
+import createFetch from '@vercel/fetch'
 import NextAuth from 'next-auth'
 import CloudIdpProvider from 'lib/auth/cloud-idp-provider'
 import refreshTokenSet from 'lib/auth/refresh-token-set'
 import isJwtExpired from 'lib/auth/is-jwt-expired'
 import { AuthErrors } from 'types/auth'
 
+const fetch = createFetch()
+
 const isDev = process.env.NODE_ENV === 'development'
 
 export default NextAuth({
 	session: {
 		maxAge: 2592000, // 30 days
 	},
+	// https://next-auth.js.org/configuration/options#events
+	events: {
+		/**
+		 * NOTE: NextAuth log out of auth providers, so we have to handle doing that
```

Comment:
💭 are we missing a word in this comment? `...NextAuth (does not?) log out of auth providers...`?

Reply:
Haha yes! Oops 🤣

Reply:
Fixed here: https://github.com/hashicorp/dev-portal/pull/1301/commits/71cb1bbedb4900fd4ce9a3ea4bdd9e734cef1046